tests/components/orderentry/Paginate.test.jsx

```diff
@@ -10,13 +10,27 @@ const props = {
   fetchNew: jest.fn(),
 }
 
+const props2 = {
+  totalPage: 193,
+  nextPageUrl: 'http://next=page=some=random=page=and=the=length=20',
+  dispatch: jest.fn(),
+  prevPageUrl: '',
+  patientId: '123eeeeeer',
+  fetchNew: jest.fn(),
+}
+
+const wrapper2 = shallow(<Paginate {...props} />)
 const wrapper = shallow(<Paginate {...props} />)
```

Comment:
Great job. However, I can't see the difference between wrapped and wrapper2.

Reply:
I made a mistake before, It has been fixed now.